OpenRA.Mods.RA2/Traits/Conditions/GrantTimedConditionOnDeploy.cs

```diff
@@ -152,7 +152,7 @@ void IResolveOrder.ResolveOrder(Actor self, Order order)
 
 			// Turn to the required facing.
 			if (Info.Facing != -1 && canTurn)
-				self.QueueActivity(new Turn(self, Info.Facing));
+				self.QueueActivity(new Turn(self, WAngle.FromFacing(Info.Facing)));
```

Comment:
To be consistent with 18228 this really should be declaring `Info.Facing` as a `WAngle` and updating the yaml to match.